### DISCUSSION TITLE
Actividad Docker - GRUPO 8

### DISCUSSION THREAD

**Discussion by tomaszbk** [frlp-utn-ingsoft/catedra#107] [Show and tell]
https://github.com/Salani1003/vetsoft

**Comment by rodriguezemautn** [comment]
Grupo 8 : 

-Su imagen tiene un peso de 1.25 Gb ---- se podria mejorar, revisen las buenas practicas.
-El comando que proporcionaron en README.md no esta en modo detach (-d) lo cual al cerrar la consola se detiene el contenedor. 
*Construye correctamente la imagen
*Despliega el contenedor
Corregir las sugerencias. 
Muy buen trabajo!

emanuel@emanuel-dell:~$ docker ps 
CONTAINER ID   IMAGE             COMMAND                  CREATED          STATUS          PORTS                                       NAMES
207418dc38f5   vetsoft-app:1.0   "/bin/sh -c 'python …"   10 minutes ago   Up 10 minutes   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp   vetsoft_vetsoft-app_1
emanuel@emanuel-dell:~$ docker images -a
REPOSITORY    TAG       IMAGE ID       CREATED          SIZE
vetsoft-app   1.0       437b97253fcd   10 minutes ago   1.25GB
emanuel@emanuel-dell:~$

**Comment by tomaszbk** [comment]
Buenas noches Ema!
Ahi agregamos el modo detach al comando, y mergeamos a main los cambios que reducen el tamano de la imagen a 310.41 MB.

Cualquier cosa decinos.
